tools/rosclean/src/rosclean/__init__.py

```diff
@@ -66,6 +66,20 @@ def quote(s):
         _call(cmds, cwd)
     return accepted
 
+def _ask(comment):
+    """
+    ask user with provided comment. If user responds with y, return True
+
+    :param comment: comment, ``str``
+    :return: ``True`` if user responds with y
+    """
+    sys.stdout.write("Okey to perform:\n\n%s\n(y/n)?\n"%comment)
+    while 1:
+        input = sys.stdin.readline().strip().lower()
+        if input in ['y', 'n']:
+            break
+    return (input == 'y')
+
```

Comment:
The same code lines are present in `_ask_and_call`. Please refactor the code to avoid the code duplication.